.github/workflows/release.yml

```diff
@@ -0,0 +1,20 @@
+name: 'release'
+on: [workflow_dispatch]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions: write-all
+    steps:
+      - uses: actions/checkout@v2
+      - uses: decentraland/gh-action-release@0.1.0
+        id: result
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - run: echo "Github context ${{ steps.result.outputs.context}}"
+      - run: echo "Last tag ${{ steps.result.outputs.lastTag}}"
+      - run: echo "Commits found ${{ steps.result.outputs.commitsLength}}"
+      - run: echo "Major being bumped ${{ steps.result.outputs.bumpMajor}}"
+      - run: echo "Minor being bumped ${{ steps.result.outputs.bumpMinor}}"
+      - run: echo "Patch being bumped ${{ steps.result.outputs.bumpPatch}}"
```

Comment:
Can we log the latest release and new release version instead? Or only log the corresponding part being bumped, for example only print "minor version being bumped"

Reply:
The are being logged as `lastTag` and `newTag`. I logged patch, minor and major only fotr debuggin purposes, we can take them out

Reply:
Okay, let's remove them then